public/common/TracySocket.cpp

```diff
@@ -651,7 +651,7 @@ void IpAddress::Set( const struct sockaddr& addr )
 #else
     auto ai = (const struct sockaddr_in*)&addr;
 #endif
-    inet_ntop( AF_INET, &ai->sin_addr, m_text, 17 );
+    inet_ntop(AF_INET, &ai->sin_addr, m_text, TEXT_SIZE);
```

Comment:
Why was this changed?

Reply:
this constant is being used in multicapture 3 times. I thought best to extract it to give it a name.

I agree TEXT_SIZE is not a good name; maybe you have a suggestion

[as other extraction changes in this MR, I can revert it - but I'll have to copy-paste its value where it's used, and that felt less clean and understandable when diving into the codebase]